metadata_backend/database/db_service.py

```diff
@@ -73,25 +73,26 @@ async def create(self, collection: str, document: Dict) -> bool:
         return result.acknowledged
 
     @auto_reconnect
-    async def exists(self, collection: str, accession_id: str) -> bool:
-        """Check object exists by its accessionId.
+    async def exists(self, collection: str, id: str) -> bool:
+        """Check object or folder exists by its generated ID.
 
         :param collection: Collection where document should be searched from
-        :param accession_id: Accession id of the document to be searched
+        :param accession_id: ID of the document or folder to be searched
```

Comment:
```suggestion
        :param accession_id: ID of the object or folder to be searched
```

Reply:
this should be actually just `:param id`